CODE_OF_CONDUCT.md

```diff
@@ -43,7 +43,7 @@ __Moderators Group:__ The moderators group is responsible for addressing immedia
 
 We begin interactions by acknowledging that we are part of a community with complementary goals. Different views are allowed to respectfully coexist in the same space. When something's happened and someone is uncomfortable, our first choice is to work through it. Endeavor to listen and appropriately adjust your behavior if someone approaches you privately with a request that you apologize or publicly requests that you stop an ongoing presentation. If someone questions your words, actions or motives, or "calls you out", hear their feedback and respond respectfully. It’s okay to not understand why something is hurtful or causes discomfort, as long as you approach it respectfully, with empathy. Repeating hurtful behavior after it has been addressed is disrespectful and is not allowed. Doing so will result in removal and subsequent banning from in-person events and being placed into moderation in online spaces. 
 
-###The first rule of engaging with others is consent
+## The first rule of engaging with others is consent
```

Comment:
Could you plesse add a new `#` here? There were 3 `#`s here before. Please add a new `#`